src/components/BarChart/BarChart.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from 'react';
+
+import Axes from './Axes';
+import Bars from './Bars';
+
+import { getMeasureArray } from './utils';
+
+import './style.css';
+
+const chartsData = [
```

Comment:
It should be passed as props

Reply:
Where from?

Reply:
You can pass any props to component using storybook